scripts/travis-emscripten/build_emscripten.sh

```diff
@@ -42,8 +42,18 @@ fi
 
 WORKSPACE=/root/project
 
-echo -en 'travis_fold:start:compiling_solidity\\r'
 cd $WORKSPACE
+
+# shellcheck disable=SC2166
+if [[ "$CIRCLE_BRANCH" = release || -n "$CIRCLE_TAG" || -n "$FORCE_RELEASE" || "$(git tag --points-at HEAD 2>/dev/null)" == v* ]]
```

Comment:
The other scripts don't seem to have the `git` part. Why add here and not there?

Reply:
I took that from a third script, and I think it is a good idea so that it also works locally.